modules/auxiliary/spoof/llmnr/llmnr_response.rb

```diff
@@ -99,9 +91,17 @@ def dispatch_request(packet, rhost, src_port)
           :address => (spoof.ipv6? ? spoof : spoof.ipv4_mapped).to_s
         )
       else
-        print_warning("#{rhost.to_s.ljust 16} llmnr - Unknown RR type, this shouldn't happen. Skipping")
+        vprint_warning("#{rhost.to_s.ljust 16} LLMNR - Skipping unhandled RR query type #{question.qType}")
         next
       end
+
+      if name =~ /#{datastore['REGEX']}/i
```

Comment:
For paranoia's sake, adding the /n flag to the Regexp may fix some corner cases.